common/sw-update/versions-db-manager.cpp

```diff
@@ -46,6 +46,7 @@ namespace rs2
             {
                 auto version_str = (*res)["version"];
                 out_version = version(version_str);
+                out_version = out_version.without_build();
```

Comment:
Why did you decide to handle it here and not in the viewer code?
Lets speak about this change and think what is best